redi-connect-front/src/pages/app/me/Mentoring.tsx

```diff
@@ -57,12 +62,14 @@ const Mentoring = ({ profile, profileSaveStart }: any) => {
 
   const { categories: selectedCategories } = formik.values
 
+  // const categoryObject = Object.assign({}, ...availableCategories.map(category => ({ [category.id]: category.label })))
+
   const readMentoring = () => {
     return <Tag.Group>
       {categories.map(
-        (categorie: any, index: number) =>
-          <Tag key={`${categorie}_${index}`} size="large" rounded>
```

Comment:
React doesn't recommend using array index as key since it changes. It should rather be a constant identifier.

Reply:
Exactly, but the index was not just the array index, so in this case the usage would be fine. I added the index because the id's will cut it...

Reply:
Don't think it's the case. The key becomes non-constant since `${categorie}_${index}` mixes a constant and a non-constant. Or no?